centos_ci.sh

```diff
@@ -88,7 +110,6 @@ function perform_artifacts_upload() {
 function get_bundle() {
   mkdir $HOME/Downloads
   curl -L 'https://github.com/code-ready/crc-ci-jobs/releases/download/4.0.1/crc_bundle.zip' -o $HOME/Downloads/bundle.zip
```

Comment:
Do we use this? If not, this is conflicting with what is in https://github.com/code-ready/crc/pull/301/files#diff-15c128a70ef5dd7aaf264bce566298fdR128 ?

Reply:
We download `bundle.zip` to `/root/Downloads`, then copy it to `home/crc_ci/Downloads` after `crc_ci` is created, then unzip it there - this results in `/home/crc_ci/crc_libvirt_4.1.3.tar.xz`.